PIF/Sources/PIFSupport/PIFSupport.swift

```diff
@@ -49,7 +49,7 @@ public class PIFCacheParser {
 
 		// If multiple workspaces exist, it's because the something in the project changed between builds. Sort workspaces by the most recent.
 		return workspaces
-			.sorted(using: KeyPathComparator(\.modificationDate))
+            .sorted(using: KeyPathComparator(\.modificationDate, order: .reverse))
```

Comment:
We should convert `PIF` to use tabs (and then adopt `swiftformat`) like the rest of the project, for now a 2 space indent to match the rest of the file.
```suggestion
        .sorted(using: KeyPathComparator(\.modificationDate, order: .reverse))
```